core/domain/exp_domain.py

```diff
@@ -68,7 +68,8 @@
 STATE_PROPERTY_INTERACTION_SOLUTION = 'solution'
 # Deprecated state properties.
 STATE_PROPERTY_CONTENT_IDS_TO_AUDIO_TRANSLATIONS_DEPRECATED = (
-    'content_ids_to_audio_translations')  # Deprecated in state schema v27.
+    # Deprecated in state schema v27.
```

Comment:
This change is done because I introduced a new lint check to prevent trailing/inline comments unless the reason is a valid allowed pragma.